news/2017-09-03-qat-gazette-september-2017.md

```diff
@@ -0,0 +1,57 @@
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+To those of you who have been following this blog at its [previous home](http://osuqat.tumblr.com/), welcome back! We’ve shuffled through a bit of a lean period on account of the summer holidays (or in my case, even more work than usual), but I am happy to bring you **the return of The QAT Gazette!** Feel free to skip down a little if you’re already familiar with our humble blog and want to get straight into the news, but if this is your first time and you’re wondering what all the fuss is about, read on!
+
+**A little over a year ago**, the Quality Assurance Team formed a blog to give people an idea of the work that goes on behind the scenes, the internal topics that come up for discussion, and our visions for shaping the future of the osu! mapping and modding community.
+
+There were highlights of upcoming projects and events, reviews of past occurrences such as Beatmap Nominator tests and the Best of 2016 awards, and sometimes opinion pieces simply giving our thoughts on various aspects of the community, the mapping meta, and so on. With the growth of the mapping and modding community over the past year, as well as the significant changes to the role of the QAT, more and more people rightfully ask, **“What the heck do you guys actually do now!?”** Hence the birth of regular updates, and now, a move into the mainstream for all to see!
+
+## Updates to the Beatmap Nomination Group
+
+Firstly, **please welcome two new Beatmap Nominators to the Advanced Tier for osu!standard - Battle and Naxess!** As you may be aware, Advanced Tier Nominators are responsible for beatmap qualification, and with the recent departure of a number of existing members, it’s important that receiving a heart icon doesn’t become a bottleneck for mappers. Please treat them well as they settle into their new role!
+
+On the other side of the coin, **a number of existing members of the Beatmap Nomination Group will be departing** to pursue other opportunities. Many thanks to Dellvangel, -Kamikaze-, Kuo Kyoka, newyams99, Nivrad00 and Sonnyc for your contributions to our community over the months and years!
+
+Fear not however, as **we are currently reviewing applications for a new batch of fresh-faced modders to replenish their ranks!** The latest application process for the osu!taiko, osu!catch and osu!mania! game modes has recently closed, with successful applicants expected to promoted in the coming weeks.
+
+With the discontinuation of Beatmap Nominator tests, new BN applications will consist of an activity check, as well as submission of the applicant’s “best” modding work - either as a standard mod post, or series of suggestions in the moddingv2 panel. 
```

Comment:
"With the discontinuation of using a test to join the Beatmap Nominators, the new application process will consist..."

will avoid using the contraction that looks ehhhhh and still make elegant sentence

Reply:
Done, also kinda convoluted wording I guess, gonna try "With Beatmap Nominator tests being recently discontinued..."